packages/app-project/docker-compose.yml

```diff
@@ -2,31 +2,41 @@ version: '3.4'
 
 services:
   dev:
-    image: front-end-monorepo_dev:latest
+    image: front-end-monorepo_project-app:local
     build:
-      context: ../../
-      target: bootstrap
-    entrypoint:
-      - "yarn"
-      - "workspace"
-      - "@zooniverse/fe-project"
-    command: ["dev"]
+      context: .
+      args:
+        ASSET_PREFIX: https://fe-project.zooniverse.org
+        SENTRY_DSN: https://2a50683835694829b4bc3cccc9adcc1b@sentry.io/1492691
+    command: [yarn, "dev"]
+    environment:
+      - PORT=3000
+      - ASSET_PREFIX= http://localhost:3000
+      - SENTRY_DSN=https://2a50683835694829b4bc3cccc9adcc1b@sentry.io/1492691
+      - NODE_ENV=development
+      - PANOPTES_ENV=staging
+      - NEXT_TELEMETRY_DISABLED=1
+      - APP_ENV=development
     ports:
       - "3000:3000"
     volumes:
+      - project_app_node_modules_cache:/usr/src/packages/app-project/node_modules
```

Comment:
`node_modules` should be installed to the root of the monorepo, not to each package individually.

Reply:
thanks - i've modified the cache path. We can remove the cache if it doesn't prove useful for quick yarn install to add / update packages vs and a full rebuild